database/email.go

```diff
@@ -7,7 +7,21 @@ import (
 	"gitlab.com/NebulousLabs/errors"
 	"go.mongodb.org/mongo-driver/bson"
 	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.sia.tech/siad/build"
+)
+
+const (
+	// EmailConfirmationTokenTTL defines the lifetime of an email confirmation
+	// token. After the token expires it can no longer be used and the user
+	// needs to request an email re-confirmation.
+	EmailConfirmationTokenTTL = 24 * time.Hour
+
+	// emailLockTTL defines how long an email can stay locked for sending. Once
+	// the lock expires the record will be unlocked and free for other servers
+	// to lock and send.
+	emailLockTTL = 5 * time.Minute
```

Comment:
This might be too long? Would 1 minute make more sense? 30 seconds?

Before, when I was using templates for the emails, I saw a template being parsed and execute for more than 600ms. This was on my local machine which is fairly fast. Imagining that a server under heavy load will want to send 10 emails in a single batch, going under 30 seconds seems unwise.